pkg/lib/cockpit-components-table.jsx

```diff
@@ -140,7 +140,7 @@ export const ListingTable = ({
             </EmptyStateBody>
         </EmptyState>;
 
-    if (rows == 0) {
+    if (rows.length == 0) {
```

Comment:
So this one doesn't solve any ignored failure from our list yet? But if we have more instances of these, this is highly useful to know indeed.

Reply:
This doesn't fix one issue because I am confused about the channel code:

The other is:

```
pkg/apps/appstream.js(54,25): error TS2367: This comparison appears to be unintentional because the types 'Error' and 'string' have no overlap.
```

Which seems fixable but I can't find the code where ProcessError would end up being "closed". So was wondering if this was old code or not?